scripts/add_translation.py

```diff
@@ -0,0 +1,115 @@
+
+""" Download all files with consent form content from google drive into csv format and combine into a single json file.
```

Comment:
See above (2x). This script does not download consent forms; its purpose is to combine the functionality of two other scripts in a convenient interface. 

The instructions that follow are helpful (since this script now handles the download functionality), but specifically this line could stand to be improved.